gcloud/logging/_gax.py

```diff
@@ -0,0 +1,252 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GAX wrapper for Logging API requests."""
+
+import json
+
+# pylint: disable=import-error
```

Comment:
`grpc` comes in via the mocked `_testing/grpc` directory, so no import failures there, if you only imported modules it'd be easy to mock but importing objects is another thing entirely